lib/registry.js

```diff
@@ -16,7 +17,9 @@ export function define(id, spec) {
   if (get(id)) {
     console.warn(`Overwriting spec ${getName(id)}`)
   }
-  spec.name = getName(id)
+  if (!spec.name && typeof spec !== "symbol") {
```

Comment:
Sorry mate, I'm not sure anymore I understand the problem we're trying to solve. What is it with the symbols vs strings? Please give me an example.